plugins/modules/lvg.py

```diff
@@ -418,89 +369,40 @@ def make_vg(module, vg_name, vg_state):
     global result
 
     pvs = module.params['pvs']
-
-    cmd = "mkvg "
+    opt = build_vg_opts(module)
 
     vg_type_opt = {
         "none": '',
         "big": '-B ',
         "scalable": '-S ',
     }
-
     vg_type = module.params["vg_type"]
     if vg_type:
-        cmd += vg_type_opt[vg_type]
-
-    enhanced_con_vg = module.params["enhanced_concurrent_vg"]
-    if enhanced_con_vg:
-        cmd += '-C '
-
-    critical_pvs = module.params["critical_pvs"]
-    if critical_pvs:
-        cmd += '-e y '
```

Comment:
At the build_vg_opts (right), if the critical_pvs is false, the cmd+=' -e n" is added. I don't see this in the older code. If that still ok?

Reply:
Answer from Joseph. It is not needed for make vg because the default is " -e n". The refactor function, in case the user adds the critical_pvs as false, it will add the '-e n' and that will actually be ignored by the command as it is the default.